v22.1/restore.md

```diff
@@ -340,6 +340,52 @@ RESTORE DATABASE bank FROM LATEST IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_
  `RESTORE` will re-validate [indexes](indexes.html) when [incremental backups](take-full-and-incremental-backups.html) are created from an older version (v20.2.2 and earlier or v20.1.4 and earlier), but restored by a newer version (v21.1.0+). These earlier releases may have included incomplete data for indexes that were in the process of being created.
 {{site.data.alerts.end}}
 
+### Restore with `AS OF SYSTEM TIME`
+
+If you did **not** run a backup with `revision_history`, it is still possible to use `AS OF SYSTEM TIME` with `RESTORE` to target a particular time for the restore. However, your restore will be limited to the times of the full backup and each incremental backup in the chain. 
```

Comment:
nit: starting with negation ("if you did not run a backup with revhist") but not having the positive use case immediately prior ("when you run a backup with revhist") seems a little sudden. What about a tiny mention like:

> Running a backup using the `revision_history` option allows you to specify any time contained within the backup chain as the point in time to recover from. However, if you did **not** run a backup with ....

Definitely optional.

Reply:
I have switched this around — good idea. It went through a few iterations, but this is the better idea 👍